tests/test_shardformer/test_model/_utils.py

```diff
@@ -124,10 +124,28 @@ def build_model_from_hybrid_plugin(
         sharded_model = copy.deepcopy(org_model)
     if use_lazy_init:
         ctx.materialize(org_model)
+    names = [
+        # "embeddings.word_embeddings.weight",
+        # "encoder.layer.0.attention.self.query.weight",
+        "encoder.layer.0.output.dense.weight",
+        "encoder.layer.0.output.dense.bias",
+        "encoder.layer.1.output.dense.weight",
+        "encoder.layer.1.output.dense.bias",
+    ]
+    # for name, param in org_model.named_parameters():
+    #     if name in names:
+    #         param.requires_grad = True
+    #     else:
+    #         param.requires_grad = False
+    # for name, param in sharded_model.named_parameters():
+    #     if name in names:
+    #         param.requires_grad = True
+    #     else:
+    #         param.requires_grad = False
 
     org_model = org_model.cuda()
-    org_optimizer = optim_class(org_model.parameters(), lr=1e-3)
-    sharded_optimizer = sharded_optim_class(sharded_model.parameters(), lr=1e-3)
+    org_optimizer = Adam(org_model.parameters(), lr=1e-3)
+    sharded_optimizer = Adam(sharded_model.parameters(), lr=1e-3)
```

Comment:
Why?

Reply:
not my code

Reply:
It seems that this code is outdated. Can you check the latest code feature/dist-optim branch?